src/root_files/app_2/index.html

```diff
@@ -49,54 +49,40 @@
                 var is_logged_in = client_info && client_info['token'];
                 window.location.href = (/\/app\//.test(window.location.pathname) ? '' : 'app/') + (lang || 'en').toLowerCase() + '/trade' + window.location.search;
             }
-            function isDelayedRedirect() {
-                if(isGtmApplicable() && !redirectTimeOut) {
-                    redirectTimeOut = setTimeout(redirect, 5000);
-                } else {
-                    clearTimeout(redirectTimeOut);
-                    redirect();
-                }
-            }
             function isGtmApplicable() {
-                return /www\.binary\.com/i.test(window.location.hostname);
+                return /(www|staging)\.binary\.(com|me)/i.test(window.location.hostname);
```

Comment:
I think we don't need `isGtmApplicable` and ` pushEvents` as we don't wait for GTM anymore

Reply:
so do we want to remove GTM from app_2/index.html completely?